sqlglot/dialects/mysql.py

```diff
@@ -27,12 +44,37 @@ class MySQL(Dialect):
     class Parser(Parser):
         STRICT_CAST = False
 
+        def _parse_ordered(self):
+            is_null = self._match(TokenType.ISNULL)
+            if is_null:
+                self._match(TokenType.L_PAREN)
```

Comment:
do you always expect l_paren ? 

self._match_l_paren()
and self._match_r_paren()